.travis.yml

```diff
@@ -6,6 +6,10 @@ cache:
   directories:
   - "$HOME/.gradle/caches/"
   - "$HOME/.gradle/wrapper/"
+  
+script:
```

Comment:
no need for this
https://docs.travis-ci.com/user/languages/java/#gradle-default-script-command